.github/template.md

```diff
@@ -165,4 +189,95 @@ We would love to hear about more ideas on how to find vulnerable infrastructure-
 
 If you need direct support you can contact us at [info@bridgecrew.io](mailto:info@bridgecrew.io).
 
-# Existing vulnerabilities (Auto-Generated)
+## Existing vulnerabilities (Auto-Generated)
```

Comment:
@AErmie This part is auto-generated, so please don't override anything after this title - just leave it as was before :)

Reply:
I'm not sure how to revert the one section. But, in the auto-generated part, you should change the heading accordingly as it causes a markdown syntax issue.